.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
What's the point of having this root config and the package configs?

Reply:
I believe that a root config is needed if I open the project from the root directory.